tests/outlineCompiler_test.py

```diff
@@ -228,6 +228,14 @@ def test_setupTable_meta(self, testufo):
         assert meta.data["PRIA"] == b"Some private ascii string"
         assert meta.data["PRIU"] == "Some private unicode string…".encode("utf-8")
 
+    def test_setupTable_name(self, FontClass):
+        font = FontClass(getpath("TestFont-name.ufo"))
```

Comment:
I see you added a whole new UFO.. that's a lot of files for just this little thing. How about you keep using the existing TestFont.ufo as the base, then load that and add the desired openTypeNameRecords to the font.lib and the compile and assert?

Reply:
Yes, I realized it was a lot of files and then pruned it, but it is still some new files. I'll adjust as you suggested.

Reply:
Updated. Please take a look.